### PR TITLE
Configure renovate, force exact package-lock versions.

### LINKED ISSUE #2409: Configure renovate
Get inspiration from addons-frontend

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This also force-replaces any http urls in package-lock to be https
again.

Fixes #2409

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,5 +1,20 @@
 {
+  "timezone": "Europe/Berlin",
```
Comment (muffinresearch) [inline renovate.json]: Any particular reason for Berlin - maybe UTC would be more universal?

File: renovate.json [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
   "extends": [
-    "config:base"
+    "config:base",
+    ":maintainLockFilesWeekly",
+    ":prNotPending"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 4pm on Tuesday"
```
Comment (wagnerand) [inline renovate.json]: Could we move this to end of Monday or early Tuesday, so we have enough time to deal with potential breakings?

Comment (willdurand) [reply renovate.json]: I think the default is on Monday morning (European time), which I find convenient.

File: renovate.json (hunk 1 above)
Comment (EnTeQuAk) [reply renovate.json]: Indeed, fixed in 7f775ffb

File: renovate.json (hunk 2 above)
Comment (EnTeQuAk) [reply renovate.json]: early Tuesday makes sense, that way we can make a new release of dispensary on Monday ideally.

Fixed in 7f775ffb

**Pull request merged**